src/main/java/ca/mcgill/ecse428/potatopeeps/order/Order.java

```diff
@@ -1,42 +1,49 @@
-package ca.mcgill.ecse428.potatopeeps.order;
-
-import ca.mcgill.ecse428.potatopeeps.diningsession.DiningSession;
-import ca.mcgill.ecse428.potatopeeps.menuitem.MenuItem;
-import lombok.Data;
-
-import javax.persistence.*;
-
-@Data
-@Entity
-public class Order {
-
-    public Order() {
-    }
-
-    public Order(Double price, Integer quantity) {
-        this.price = price;
-        this.quantity = quantity;
-    }
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private Long id;
-
-    @Column(name = "price")
-    private Double price;
-
-    @Column(name = "status")
-    private String status;
-
-    @Column(name = "quantity")
-    private Integer quantity;
-
-    @ManyToOne
-    @JoinColumn(name = "menu_item_id", nullable = false)
-    private MenuItem menuItem;
-
-    @ManyToOne
-    @JoinColumn(name = "dining_session_id", nullable = false)
-    private DiningSession diningSession;
-
-}
+package ca.mcgill.ecse428.potatopeeps.order;
+
+import ca.mcgill.ecse428.potatopeeps.diningsession.DiningSession;
+import ca.mcgill.ecse428.potatopeeps.menuitem.MenuItem;
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+public class Order {
+
+    public Order() {
+    }
+
+    public Order(Double price, Integer quantity) {

```

Comment:
Is this constructor necessary? I.e. can/should an order ever be created without an associated menuItem?

Reply:
To add on to this, an order shouldn't be created without an associated dining session, no?

Reply:
Depends on how we want to go about dining sessions. Say we clear a table (pay bill) this sets the status of that dining session to finished. The next dining session could be created in the constructor of an order. E.g. if no existing live dining session, create one and assign this order to that dining session